rule/templates/_rule.js

```diff
@@ -8,28 +8,37 @@
 // Rule Definition
 //------------------------------------------------------------------------------
 
-module.exports = function(context) {
+module.exports = {
+    meta: {
+        docs: {
+            description: "<%= desc %>",
+            category: "Fill me in",
+            recommended: false
+        },
+        fixable: null,  // or "code" or "whitespace"
+        schema: [
+            // fill in your schema
+        ]
+    },
 
-    // variables should be defined here
+    create: function(context) {
 
-    //--------------------------------------------------------------------------
-    // Helpers
-    //--------------------------------------------------------------------------
+        // variables should be defined here
 
-    // any helper functions should go here or else delete this section
+        //----------------------------------------------------------------------
+        // Helpers
+        //----------------------------------------------------------------------
```

Comment:
I thought about that too, but ultimately, it depends on whether you need access to `context` from inside helpers. And we can't predict that, so it's safe to have helpers inside `create` function